ListApp/ListApp.Api.Tests/Utils/ListItemEqualityComparer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using ListApp.Api.Models;
+
+namespace ListApp.Api.Tests.Utils
+{
+    /// <summary>
+    /// Equality comparer used for testing equality of <see cref="ListItem"/> models
+    /// It does NOT implement its own GetHashCode() method - use ONLY
+    /// for comparing two <see cref="ListItem"/> models!
+    /// </summary>
+    public class ListItemEqualityComparer : IEqualityComparer<ListItem>
```

Comment:
That is all nice until you realize how you use it
* every time, in every test you create a new instance of the very comparer that consists of 2 pure functions → you should only use one static instance of the class (it is actually perfect candidate for a singleton)
* to move the singleton pattern a bit further, you will end-up with repeating `….using(…StaticComparerInstance)` → to make things more readable, why not introduce an extension method (call, e.g. `usingListItemComparer()`) that would be easy to find by IntelliSense and describe what's going on perfectly (→ not the file would best fit into a folder called `Extensions`)